src/microbe_stage/Microbe.cs

```diff
@@ -1412,6 +1459,37 @@ public override void _IntegrateForces(PhysicsDirectBodyState physicsState)
         physicsState.Transform = GetNewPhysicsRotation(physicsState.Transform);
     }
 
+    /// <summary>
+    ///   Remake the colony, only needed after load
+    /// </summary>
+    public void RemakeColony()
+    {
+        var col = new List<Microbe>(ColonyChildren);
+        var sem = 0;
+
+        // Check if the pili of colony members have been formed
+        foreach (var member in col)
+        {
+            foreach (var organelle in member.organelles)
+                if (organelle.Definition.Name == "Predatory Pilus" && member.pilusPhysicsShapes.Count == 0)
```

Comment:
https://github.com/Revolutionary-Games/Thrive/pull/2379#discussion_r667486970

Reply:
Lol yea, ill fix it.

Reply:
Is that the only  code problem u saw?